packages/expo-notifications/android/src/main/java/expo/modules/notifications/serverregistration/InstallationId.java

```diff
@@ -25,6 +25,7 @@
 
   public static final String LEGACY_UUID_KEY = "uuid";
   public static final String UUID_FILE_NAME = "expo_installation_uuid.txt";
+  public static final String SCOPED_ID_FILE_NAME = "expo_notifications_installation_id.txt";
```

Comment:
this is a small nit, but we should be consistent about using ID vs UUID in these variable & file names. so maybe change L28 to 
```
  public static final String SCOPED_UUID_FILE_NAME = "expo_notifications_installation_uuid.txt";
```

Reply:
Good nit, for a reason that made sense to me then I decided to drop the UUID and from now on use "ID" because why would a file name need to specify what type of ID it holds?

I'm ok with either following your suggestion or changing `UUID_FILE_NAME` to `ID_FILE_NAME` (too). What do you think?

Reply:
Changed to `UUID`! 👍